client/webserver/site/src/js/markets.ts

```diff
@@ -1100,8 +1106,8 @@ export default class MarketsPage extends BasePage {
     this.marketList.select(host, base, quote)
     this.setLoaderMsgVisibility()
     this.setTokenApprovalVisibility()
-    this.setRegistrationStatusVisibility()
-    this.resolveOrderFormVisibility()
+    await this.setRegistrationStatusVisibility()
+    await this.resolveOrderFormVisibility()
```

Comment:
Why these `await` changes? What were you seeing?

Reply:
I've removed them. I just added them because `resolveOrderFormVisibility` became `async`, but it's fine without them.